hyped.code-workspace

```diff
@@ -13,11 +13,13 @@
     "files.autoSave": "onWindowChange",
```

Comment:
Is it possible to maybe have this file moved somewhere out of the root directory and maybe into `utils/` or something?

Reply:
We [discussed this before](https://github.com/Hyp-ed/hyped-2020/pull/10#issuecomment-555437995) :smile: 
(Also, I think it would belong under a separate PR if we were to move it.)

Reply:
Yea I remember that, but it would be nice if we could tidy up our root directory a bit. If it's just a matter of updating the relative paths it doesn't sound like too much too change imo.

Reply:
Not sure if it's just that. We'd need to test it. Still, it's not a good practice to just tack unrelated stuff onto a PR. Better to have a tiny separate one. Do you want to add a task for it on ClickUp?